husky_control/config/localization.yaml

```diff
@@ -1,6 +1,6 @@
-odom_frame: odom
+odom_frame: world
```

Comment:
Can you leave this one as is and make a new file called `localization_sim.yaml` and change the corresponding launch files?